src/main/java/com/google/devtools/build/lib/rules/python/PyBuiltins.java

```diff
@@ -266,44 +266,6 @@ public Object expandLocationAndMakeVariables(
         .expand(attributeName, expression);
   }
 
-  // TODO(b/232136319): Remove this once the --experimental_build_transitive_python_runfiles
-  // flag is flipped and removed.
-  @StarlarkMethod(
-      name = "new_empty_runfiles_with_middleman",
```

Comment:
Why are you removing these methods? It looks like it's an incompatible change (which may still be fine; I am not up to date on the details of our Python rules)